packages/at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -505,7 +507,8 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
       ..regex = _atClient.getPreferences()!.syncRegex
       ..limit = _atClient.getPreferences()!.syncPageLimit
       ..isPaginated = true;
-    _logger.finer('** syncBuilder ${syncBuilder.buildCommand()}');
+    _logger.finer(
+        '${_atClient.getPreferences()!.atClientParticulars.clientId} | syncBuilder ${syncBuilder.buildCommand()}');
```

Comment:
Why not add a client-only extension to AtSignLogger as you have done in at_server?

Reply:
Sure Gary, will update this it is in at_server and push the changes.